torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -542,25 +543,25 @@ XLAGraphExecutor::SyncTensorCollection XLAGraphExecutor::CollectSyncTensors(
       }
     }
   }
-  // Mix the hash with the resource domain hashes as compile handles are only
-  // valid within a domain (usually a single host).
+  // Mix the hash with the resource domain hashes as compile handles are
+  // only valid within a domain (usually a single host).
   coll.hash = torch::lazy::MHash(
       coll.hash,
       xla::ComputationClient::Get()->GetResourceDomain(coll.device.toString()));
   if (!at_tensors.empty()) {
     TORCH_LAZY_COUNTER("SyncTensorsToData", at_tensors.size());
     // Create data handles with shardings. If a tensor has a
     // sharding annotation, then a BackendDataPtr with PjRtShardedData is
-    // returned; if there is no sharding annotation, then a BackendDataPtr with
-    // PjRtData is returned.
+    // returned; if there is no sharding annotation, then a BackendDataPtr
```

Comment:
any chance we can undo these comment changes? They make reviewing a bit difficult

Reply:
Removed unnecessary linting changes.